templates/home/index.html.twig

```diff
@@ -1,7 +1,54 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}{{ someVariable }}{% endblock %}
+{% block title %}Projektai{% endblock %}
 
 {% block body %}
-    <div class="nfq-academy">{{ someVariable }}</div>
+    <ul class="list-group m-4">
+        <li class="list-group-item list-group-item-info">Studentai</li>
+        {% for project in students %}
+            <li class="list-group-item">
+                <a href="/student?name={{ project.student | url_encode }}&project={{ project.project | url_encode }}">
```

Comment:
```suggestion
                <a href="{{ path('student', {'name': project.student, 'project': project.project}) }}">
```
Gi esi apsirašęs ` @Route("/student", name="student")`, tai ir naudok nuorodoms generuoti skirtą funkciją (`path`).
Ji savyje jau turi `url_encode` filtrą).

Plačiau:
 * https://symfony.com/doc/4.2/templating.html#linking-to-pages

Reply:
Testai nepraeina: https://github.com/aurelijusb/kickstart/pull/130/checks?check_run_id=327946096:
```
  1) First homework Have properly escaped student:
     AssertionError: Expected url_encode Twig filter: actual: 
http://nginx.symfony/student?name=%3Cb%3EIr%3C%2Fb%3E%20jo%20%22geras%22%20draug%27as&project=hack%3Cb%3Eer%3C%2Fb%3E%27is%20po%20.mySubdomain%20%26project%3D123
 does not ends with 
?name=%3Cb%3EIr%3C/b%3E%20jo%20%22geras%22%20draug%27as&project=hack%3Cb%3Eer%3C/b%3E%27is%20po%20.mySubdomain%20%26project%3D123
 See https://symfony.com/doc/4.2/templating.html#linking-to-pages and https://twig.symfony.com/doc/2.x/filters/url_encode.html
```
http://hw1.nfq2019.online/student?name=%3Cb%3EIr%3C%2Fb%3E%20jo%20%22geras%22%20draug%27as&project=hack%3Cb%3Eer%3C%2Fb%3E%27is%20po%20.mySubdomain%20%26project%3D123
:vs:
http://hw1.nfq2019.online/student?name=%3Cb%3EIr%3C/b%3E%20jo%20%22geras%22%20draug%27as&project=hack%3Cb%3Eer%3C/b%3E%27is%20po%20.mySubdomain%20%26project%3D123

Nes skirasi rezuktatai tarp:
`path(..., {...})`
:vs: 
`url_encode`

Reply:
Pataisiau savo testus: https://github.com/aurelijusb/kickstart/pull/130/checks?check_run_id=327973558

Dėkui, kad parodei alternatyvų būdą :+1: